src/systems/music/exploration_music.rs

```diff
@@ -1,15 +1,29 @@
+use crate::resources::music::Music;
 use bevy::{
-    audio::{PlaybackMode, PlaybackSettings, Volume},
-    prelude::{AssetServer, AudioBundle, Commands, Res},
+    asset::AssetServer,
+    audio::{AudioBundle, PlaybackMode, PlaybackSettings, Volume},
+    prelude::{Commands, Res},
 };
 
-use crate::resources::music::Music;
-
 pub fn play_exploration_music(
     mut commands: Commands,
-    asset_server: Res<AssetServer>,
     music: Res<Music>,
+    asset_server: Res<AssetServer>,
+    // mut spawn_audio_event: EventWriter<SpawnAudioEvent>,
 ) {
+    // let entity = commands.spawn_empty().id();
```

Comment:
Or this also not working

Reply:
probably  need to expand spawn audio to let it handle scenarios where there is no entity to attach to